### PR TITLE
fix: async build progress table wording

### PR DESCRIPTION
**Pull request by nomagick**

### CODE REVIEW

File: hubble/executor/hubio.py [hunk 1]
```diff
@@ -840,24 +840,25 @@ def _prettyprint_status_usage(
         )
 
         param_str.add_column(
-            'If you don\'t want to wait for the Executor\'s build status, you can use \'Ctrl + C\'. '
-            'After that you can get the status of the Executor any time using the command.'
+            'If you don\'t want to wait for the build result, feel free to interrupt with \'C-C\'. '
```
Comment (mapleeit) [inline hubble/executor/hubio.py:843]: Can `C-C` be understood by users? Or just a typo here.

**Review by mapleeit — CHANGES_REQUESTED** [review]

File: hubble/executor/hubio.py (hunk 1 above)
Comment (nomagick) [reply hubble/executor/hubio.py:843]: On PC it's `Ctrl` but on Mac probably `Command`.

**Review by mapleeit — APPROVED** [review]
LGTM!

File: hubble/executor/hubio.py [hunk 2]
```diff
@@ -840,24 +840,25 @@ def _prettyprint_status_usage(
         )
 
         param_str.add_column(
-            'If you don\'t want to wait for the Executor\'s build status, you can use \'Ctrl + C\'. '
-            'After that you can get the status of the Executor any time using the command.'
+            'If you don\'t want to wait for the build result, feel free to interrupt with \'Ctrl + C\'. '
+            'You can resume following the build progress any time.'
```
Comment (alexcg1) [inline hubble/executor/hubio.py:844]:
```suggestion
            'If you don\'t want to wait for the build result, you can interrupt it with \'Ctrl + C\'. '
            'You can resume the build progress at any time.'
```

File: hubble/executor/hubio.py [hunk 3]
```diff
@@ -840,24 +840,25 @@ def _prettyprint_status_usage(
...
         )
         param_str.add_row(
-            f'Use `jina hub status {work_path}` to get the last build status of the Executor',
+            f'Do `jina hub status {work_path}` to check for latest build status of the '
+            'Executor in current working directory',
```
Comment (alexcg1) [inline hubble/executor/hubio.py:848]:
```suggestion
            f'Run `jina hub status {work_path}` to check the latest build status of the '
            'Executor in the current working directory',
```

File: hubble/executor/hubio.py [hunk 4]
```diff
@@ -840,24 +840,25 @@ def _prettyprint_status_usage(
...
         )
 
         if task_id:
             param_str.add_row('')
             param_str.add_row(
-                f'You can also use `jina hub status --id {task_id}` '
-                'to get the specified build status of the Executor.',
+                f'You can also do `jina hub status --id {task_id} --replay --verbose` '
+                'to get the full log of this particular build.',
```
Comment (alexcg1) [inline hubble/executor/hubio.py:855]:
```suggestion
                f'You can also run `jina hub status --id {task_id} --replay --verbose` '
                'to get the full log of this particular build.',
```

**Review by alexcg1 — CHANGES_REQUESTED** [review]

**Comment by nomagick** [comment]
Thanks @alexcg1

**Pull request merged**